.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .idea
 .env
-src/key.json
\ No newline at end of file
+src/key.json
```

Comment:
Agreed, but I’d say it’s not relevant for this pull request. As we’ve discussed before, the two now projects will be merged into one, so that’ll happen pretty soon, probably in the next pull request I submit